modules/imaging_uploader/jsx/ImagingUploader.js

```diff
@@ -100,26 +230,28 @@ class ImagingUploader extends React.Component {
           </div>
           <div id="mri_upload_table">
             <StaticDataTable
-              Data={this.state.Data.Data}
-              Headers={this.state.Data.Headers}
-              getFormattedCell={formatColumn}
+              Data={this.state.data.Data}
+              Headers={this.state.data.Headers}
+              hiddenHeader={this.state.hiddenHeaders}
```

Comment:
i don't see this prop used in StaticDataTable. is there a PR that adds it?

Reply:
I think the PR which added it was sent to major.

Reply:
Yes, but it was pushed to major #3967. I can remove this until minor is merged into major (or major into minor? I can't keep track).